backend/src/main/resources/application.yml

```
@@ -0,0 +1,9 @@
+spring:
+  profiles:
+    active: local
+api:
+  tmap:
+    base-url: https://apis.openapi.sk.com/tmap
+    api-key:
+    endpoint:
+      route-optimization-20: /routes/routeOptimization20
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)